src/api/index.ts

```diff
@@ -139,6 +139,41 @@ export function fetchCommits(params: FileParams) {
     });
 }
 
+export async function fetchRawDataFile(params: FileParamsWithSHA) {
```

Comment:
this duplicates a lot of `fetchDataFile`, could `fetchDataFile` call `fetchRawDataFile` then parse the result?

Reply:
@jaked of course!